boa3_test/examples/nep17.py

```diff
@@ -254,17 +254,20 @@ def onNEP17Payment(from_address: UInt160, amount: int, data: Any):
 
     :param from_address: the address of the one who is trying to send cryptocurrency to this smart contract
     :type from_address: UInt160
-    :param amount: the amount of cryptocurrency that is being sent to the this smart contract
+    :param amount: the amount of cryptocurrency that is being sent to this smart contract
     :type amount: int
     :param data: any pertinent data that might validate the transaction
     :type data: Any
     """
+    if from_address is None:
+        return
+    from_addr = cast(UInt160, from_address)
```

Comment:
ideally this cast should not be necessary as from the 2 lines above the compiler could learn that `from_address` on line 268 and 271 can never be `None`. I also had to change the name due to https://app.clickup.com/t/339zp6m